src/grandorgue/GOOrganController.cpp

```diff
@@ -123,6 +123,7 @@ GOOrganController::GOOrganController(
     m_MainWindowData(this, wxT("MainWindow")) {
   GOOrganModel::SetMidiDialogCreator(pMidiDialogCreator);
   GOOrganModel::SetModelModificationListener(this);
+  m_setter = new GOSetter(this);
```

Comment:
In your case the `GOSetter` are created twice: in the GOOrganController constructor and in the `ReadOrganFile` (line 252).

So if you add initialising m_setter here, please, remove it from the line 252